src/ui/variables.css

```diff
@@ -55,6 +57,11 @@
     --border-color: lch(from var(--panel-bg-color) calc(l + 15) c h);
 
     --text-color-primary: white;
+
+    --button-ghost-bg-default: rgba(255, 255, 255, 0);
```

Comment:
Given those are shades, wondering if we can infer them from a color instead of hardcoding.

Reply:
great idea. done!